docs/en/05_Contributing/01_Code.md

```diff
@@ -71,6 +71,39 @@ We ask for this so that the ownership in the license is clear and unambiguous, a
 The core team is then responsible for reviewing patches and deciding if they will make it into core.  If
 there are any problems they will follow up with you, so please ensure they have a way to contact you!
 
+
+### Picking the right version
+
+SilverStripe core and module releases (since the 3.1.8 release) follow the [Semantic Versioning](http://semver.org) 
+(SemVar) specification for releases. Using this specification declares to the entire development community the severity 
+and intention of each release. It gives developers the ability to safely declare their dependencies and understand the
+scope involved in each upgrade.
+
+Each release is labeled in the format `$MAJOR`.`$MINOR`.`$PATCH`. For example, 3.1.8 or 3.2.0.
+
+* `$MAJOR` version is incremented if any backwards incompatible changes are introduced to the public API. 
+* `$MINOR` version is incremented if new, backwards compatible **functionality** is introduced to the public API or 
+	improvements are introduced within the private code. 
+* `$PATCH` version is incremented if only backwards compatible **bug fixes** are introduced. A bug fix is defined as 
+	an internal change that fixes incorrect behavior.
+
+**Public API** refers to any aspect of the system that has been designed to be used by SilverStripe modules & site developers. In SilverStripe 3, because we haven't been clear, in principle we have to treat every public or protected method as *potentially* part of the public API, but sometimes it comes to a judgement call about how likely it is that a given method will have been used in a particular way. If we were strict about never changing publicly exposed behaviour, it would be difficult to fix any bug whatsoever, which isn't in the interests of our user community.
+
+In future major releases of SilverStripe, we will endeavour to be more explicit about documenting the public API.
+
+**Contributing bug fixes**
+
+Raise your pull request against the same $MAJOR branch as your project. For example, if your project is on 3.3.1, raise it against the 3 branch. It will be included in the next minor release, e.g. 3.4.0. You should then upgrade your project to 3.4.0 when it is released.
```

Comment:
Bug fixes should be raised against the `$MINOR` branch: if you’re trying to fix a bug in 3.3.1, it should be raised against the 3.3 branch so it will be included in 3.3.2